coursedata/texts/es.yaml

```diff
@@ -43,14 +43,14 @@ ui:
     go_back_to_main: "Volver a la página principal"
     no_such_level: "No se encontró ese nivel de Hedy"
     no_such_class: "No se encontró esa clase de Hedy"
-    invalid_class_link: "Enlace incorrecta para unirse a la clase"
+    invalid_class_link: "Enlace incorrecto para unirse a la clase"
     no_such_program: "No se encontró el programa de Hedy"
-    invalid_teacher_invitation_code: "El código de instructor es inválida. Para ser instructor, por favor mande un mensaje a hedy@felienne.com."
+    invalid_teacher_invitation_code: "El código de instructor es inválido. Para ser instructor, por favor mande un mensaje a hedy@felienne.com."
     page_not_found: "¡No pudimos encontrar esta página!"
     not_teacher: "Parece que no eres un instructor."
     not_user: "Parece que hay que registrarse o ingresar a tu cuenta"
     not_enrolled: "No se encontró tu registración a esta clase"
```

Comment:
Here there is a small mistake: "registración" doesn't exist in Spanish, the translation for "registration" is "registro".

Reply:
¡Gracias!